packages/graphql-relational-schema-transformer/src/RelationalDBTemplateGenerator.ts

```diff
@@ -60,7 +59,7 @@ export class RelationalDBTemplateGenerator {
    * @returns the json, string form of the template given.
    */
   public printCloudformationTemplate(template: Template): string {
-    return cloudform(template);
+    return JSON.stringify(template, undefined, 2);
```

Comment:
The package `cloudform` is all the same as `cloudform-types` except for an additional `JSON.stringfy` for the template. https://github.com/bright/cloudform/blob/master/packages/cloudform/index.ts